readme.md

```diff
@@ -0,0 +1,23 @@
+# Welcome 
+
+Welcome to the Courtgrabber Documentation public repository. These files have been formatted according to the MkDocs theme and are best viewed on an MkDocs site. We have not yet found a permanent hosting service, but we hope to make the move soon. In the meantime, you can view the files by downloading MkDocs, cloning this repo, and running the built-in MkDocs dev server to view the site locally.
+
+## Installing MkDocs
+
+Run the following command from your command line to install MkDocs:
+
+`pip install mkdocs`
+
+If you encounter any issues, refer to the [MkDocs Installation Guide](https://www.mkdocs.org/user-guide/installation/) for more detailed instructions.
+
+## Creating your Courtgrabber project
+
+[Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repo to your local machine. Your project directory should contain the `mkdocs.yml` configuration file and a `docs` directory with four Markdown files and an `images` subdirectory.
```

Comment:
I'd avoid mentioning `four`.  If that number were to change in the future, this file will need to be updated as well.

Reply:
Okay. I changed it to "several."